### ISSUE TITLE
[@types/node-fetch] Blob type should accept constructor parameters

### ISSUE THREAD

**Issue by Pajn** [DefinitelyTyped/DefinitelyTyped#34612]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/node-fetch` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @torstenwerner @nikcorg @vinaybedre @kyranet @AndrewLeedham @JasonLi914

If you do not mention the authors the issue will be ignored.

Both the documentation for the Blob type and the node-fetch specific implementation accepts arguments to the constructor 
https://github.com/bitinn/node-fetch/blob/master/src/blob.js
https://developer.mozilla.org/en-US/docs/Web/API/Blob

Also the size and type properties are read only in both the documentation and the implementation.

Nothing of this is reflected in the types.

**Comment by Pajn** [comment]
Even worse, the Blob type isn't actually exported from `node-fetch` which the type implies and tricks you into using it.

Blob should be an interface and not a class...

**Comment by nikcorg** [comment]
Can you rephrase into a question.
